recipes/redis/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = '3.2.0' %}
+
+package:
+  name: redis
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/antirez/redis.git
+  git_rev: "{{ version }}"
+
+build:
+  number: 0
+  skip: true  # [win]
+  script_env:
+    - "MAJOR_MINOR_VERSION={{ '.'.join(version.split('.')[0:2]) }}"
+
+test:
+  requires:
+    - tk 8.5*
+  commands:
+    - "cd {{ SRC_DIR }} && tclsh8.5 tests/test_helper.tcl"
```

Comment:
Maybe this should be in `build.sh`. Seems more natural than switching back to the source directory here.